.github/workflows/pd-store-ci.yml

```
@@ -87,6 +89,16 @@ jobs:
           cp $HOME/.m2/settings.xml /tmp/settings.xml
           mv -vf .github/configs/settings.xml $HOME/.m2/settings.xml
 
+      - name: Install deps
+        run: |
+          ${TRAVIS_DIR}/install-deps.sh
+
+      - name: Package
+        # todo remove --fail-at-end after test
+        run: |
```

actionlint (1.7.8)

shellcheck reported issue in this script: SC2086:info:2:8: Double quote to prevent globbing and word splitting

(shellcheck)